src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,375 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.exception.ErrorMessage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+
+    private static Command commandType;
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String FILE_PATH = "data/duke.txt";
+
+    public static void addEventTaskFromInput(String command) {
+        try {
+            String description = getDeadlineAndEventDescription(command, "/at");
+            String time = getDeadlineAndEventTime(command, "/at");
+            readEventTaskToArray(description, time);
+            printAddMessage(tasks.get(tasks.size() - 1));
+        } catch (DukeException e) {
+            ErrorMessage.printUnknownError();
+        }
+    }
+
+    public static void addDeadlineTaskFromInput(String command) {
+        try {
+            String description = getDeadlineAndEventDescription(command, "/by");
+            String time = getDeadlineAndEventTime(command, "/by");
+            readDeadlineTaskToArray(description, time);
+            printAddMessage(tasks.get(tasks.size() - 1));
+        } catch (DukeException e) {
+            System.out.println("ERROR");
```

Comment:
Consider specifying the different error types! For debugging purposes, and also for user friendliness